src/full/Agda/Interaction/Options.hs

```diff
@@ -369,6 +368,7 @@ checkOpts opts
   | otherwise = return opts
   where
   matches = length . filter ($ opts)
+  optionChanged :: Eq a => (CommandLineOptions -> a) -> Bool
```

Comment:
Note that this line is removed in #5082 (https://github.com/agda/agda/pull/5082/files#diff-0d430ca0ebe120e7be80404effdd33e745a95713f9e0c76ca38fda01178098b8L372) and so will cause a trivial conflict there. I'll merge this first and resolve it there.